src/dmd/backend/elfobj.d

```diff
@@ -241,8 +239,14 @@ enum
 __gshared
 {
 
-// Symbol Table
-Outbuffer  *SYMbuf;             // Buffer to build symbol table in
+struct ElfObj
+{
+    // Symbol Table
+    Barray!Elf32_Sym SymbolTable;
+    Barray!Elf64_Sym SymbolTable64;
```

Comment:
Shouldn't this be an `union` ?

Reply:
I think it's okay to waste a bit of space here (there's only one global) for the sake of debuggability. However, I'll note that an anonymous union would need less typing :).